apps/gitness/src/pages/repo/repo-summary.tsx

```diff
@@ -74,23 +68,17 @@ export const RepoSummary: React.FC = () => {
     queryParams: { include_commit: true, git_ref: normalizedGitRef }
   })
 
-  const repoDetailsContent = repoDetails?.content
-  const repoEntryPathToFileTypeMap = useMemo(
-    (): Map<string, OpenapiGetContentOutput['type']> =>
-      new Map(
-        // @ts-expect-error remove "@ts-expect-error" once type issue for "content" is resolved
-        repoDetailsContent?.content?.entries.map(entry => [entry.path, entry.type])
-      ),
-    // @ts-expect-error remove "@ts-expect-error" once type issue for "content" is resolved
-    [repoDetailsContent?.content?.entries]
-  )
+  const repoEntryPathToFileTypeMap = useMemo((): Map<string, OpenapiGetContentOutput['type']> => {
+    if (repoDetails?.content?.entries?.length === 0) return new Map()
```

Comment:
this response actually contains the keyword `content`?

Reply:
Yes, it's defined as below

`export interface OpenapiGetContentOutput {
    content?: OpenapiContent;
    latest_commit?: TypesCommit;
    name?: string;
    path?: string;
    sha?: string;
    type?: OpenapiContentType;
}`